quickstarts/c#/PersonalizerExample/Program.cs

```diff
@@ -1,19 +1,24 @@
-﻿using Microsoft.Azure.CognitiveServices.Personalizer;
+﻿// <Dependencies>
+using Microsoft.Azure.CognitiveServices.Personalizer;
 using Microsoft.Azure.CognitiveServices.Personalizer.Models;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+// </Dependencies>
 
 namespace PersonalizerExample
 {
     class Program
     {
-        // The key specific to your personalizer service instance; e.g. "0123456789abcdef0123456789ABCDEF"
-        private const string ApiKey = "";
+        // <classVariables>
+        // The key specific to your personalizer resource instance; e.g. "0123456789abcdef0123456789ABCDEF"
+        private const string ApiKey = "9e0684f792f94b858d211e49de169474";//Environment.GetEnvironmentVariable("PERSONALIZER_RESOURCE_KEY");
```

Comment:
Looks like you accidentally submitted your key. I suggest removing this and rotating your keys.

Reply:
Thank you for letting me know. Removed those.